README.md

```diff
@@ -11,31 +11,39 @@ Run safety benchmarks against AI models and view detailed reports showing how we
 
 ## Background
 
-This is a [MLCommons project](https://mlcommons.org/ai-safety), part of the [AI Safety Working Group](https://mlcommons.org/working-groups/ai-safety/ai-safety/).
-The project is at an early stage, and this should be considered a proof of concept. Results are not intended to
-indicate actual levels of AI system safety. You can see sample benchmarks [here](https://mlcommons.org/benchmarks/ai-safety/) and read the white paper [here](https://arxiv.org/abs/2404.12241).
-
-ModelBench is closely related to [ModelGauge](https://github.com/mlcommons/modelgauge). ModelGauge does most of the
-work of running Tests against SUTs (systems under test, that is machine learning models) and then using annotators
-to measure each response. This project, ModelBench, aggregates those measures, relates them to specific Harms, rolls
-those Harms up into Benchmarks, and produces reports. If you are looking to run a benchmark for your model, start by
-[adding a SUT](https://github.com/mlcommons/modelgauge/blob/main/docs/tutorial_suts.md) that works with ModelGauge. 
+This is a [MLCommons project](https://mlcommons.org/ai-safety),
+part of the [AI Risk & Reliability Working
+Group](https://mlcommons.wpenginepowered.com/working-groups/airr/ai-risk-and-reliability/).
+The project is at an early stage. You can see sample benchmarks
+[here](https://ailuminate.mlcommons.org/benchmarks/).
+
+This project now contains both ModelGauge and ModelBench. ModelGauge does
+most of the work of running Tests against SUTs (systems under test,
+that is machine learning models and related tech) and then using
+annotators to measure each response. ModelBench aggregates those measures,
+relates them to specific Harms, rolls those Harms up into Benchmarks, and
```

Comment:
Don't we use the terminology "Hazard" instead of harm?

Reply:
We do! Thanks for catching that.